### PR TITLE
chore: storybook update node version dockerfile

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

Update node version for yarn 2.0

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> Programs must be written for people to read, and only incidentally for machines to execute

File: Storybook [hunk 1]
```diff
@@ -12,16 +12,21 @@ RUN yarn install --immutable
...
+
 # Build the app
 RUN yarn build-storybook
 
 # Bundle static assets with nginx
 FROM nginx:latest
+
 # Copy built assets from builder
-COPY --from=builder /app/dist/storybook/* /usr/share/nginx/html
+COPY --from=builder /app/dist/storybook/\* /usr/share/nginx/html
```
Comment (ctjhoa) [inline Storybook:23]: Why escape `*` ?

Comment (RemiBonnet) [reply Storybook]: it's my prettier I don't know why, I remove it and just change the the node version

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**